base/number.jl

```diff
@@ -133,11 +133,41 @@ zero{T<:Number}(::Type{T}) = convert(T,0)
 """
     one(x)
 
-Get the multiplicative identity element for the type of `x` (`x` can also specify the type
-itself). For matrices, returns an identity matrix of the appropriate size and type.
+Return a multiplicative identity for `x` (which can be either a value
```

Comment:
"multiplicative identity" may not be clear to everyone, better to somewhere say "such that `one(x)*x == x*one(x) == x`".